cpu/native/include/socket_zep.h

```diff
@@ -13,6 +13,16 @@
  *
  * @see @ref net_zep for protocol definitions
  *
+ * This ZEP implementation can send a dummy HELLO packet on startup.
+ * This is used to make dispatchers aware of the node.
+ * To enable this behavior, add
```

Comment:
Hi, I am an implementer of a custom ZEP dispatcher. What are those HELLO dummy packets? How does the dispatcher recognize them as such? Does this belong to the ZEP standard? I don't find any information on those HELLO dummy packets. ;-)

Reply:
Well the dispatcher doesn't have to care -  my initial idea was for them to not have a ZEP header at all, so the dispatcher could just drop any non-ZEP traffic. If the dispatcher didn't care, it would be dropped latest by the ZEP implementation.

How if the dispatcher does parse the ZEP header to model more complex typologies, it also needs to know about the HELLO packet - I'll add a note.

Reply:
Some specs on the ZEP header how it currently looks like would be nice either way ;-).

Reply:
Just for the HELLO or the ZEP header in general?

Reply:
Just the HELLO packet should be enough.

Reply:
like so 6dd987ea7dd0298a020710c5c859a02a4a1deae1?